src/utils.js

```diff
@@ -9,11 +9,11 @@ export function ensureLuxProp( context ) {
 	return __lux;
 }
 
-export function* entries( obj ) {
+export function *entries( obj ) {
```

Comment:
while this may work, the correct syntax is `function* entries` not `function *entries`